rst_files/getting_started.rst

```diff
@@ -128,37 +74,18 @@ Either way you should now be looking at something like this (modulo your operati
 
     .. code-block:: none
 
-        ] add IJulia InstantiateFromURL Revise REPL
+        ] add IJulia InstantiateFromURL; precompile
```

Comment:
Changed this to `:class: no-execute` Julia code per formatting suggestions from Matt.